README.md

```diff
@@ -97,7 +102,34 @@ The SDO owner services are packaged as a single docker container that can be run
    ```bash
    curl -sS -w "%{http_code}" -X POST $SDO_RV_URL/mp/113/msg/20 | jq
    ```
+   
+### <a name="gen-keypair"></a>Generate Owner Key Pairs
+
+If you want to expedite the process of creating key pairs, you can run `keys/generate-key-pair.sh`
+To run this script you must be using Ubuntu. 
+
+1. Go to the directory where you want your generated keys to be saved then download `generate-key-pair.sh`, which is used to create key pairs for Owner Attestation:
```

Comment:
Since i suggested more of an overview in the sentence above, you could remove ", which is used to create key pairs for Owner Attestation"